redash/query_runner/uptycs.py

```diff
@@ -0,0 +1,136 @@
+        auth_var = jwt.encode({'iss': key}, secret, algorithm='HS256')
+        authorization = "Bearer %s" % (auth_var)
+        header['date'] = date
+        header['Authorization'] = authorization
+        return header
+
+    def transformed_to_redash_json(self, data):
+        transformed_columns = []
+        rows = []
+        # convert all type to JSON string
+        # In future we correct data type  mapping later
+        if 'columns' in data:
+            for json_each in data['columns']:
+                name = json_each['name']
+                new_json = {"name": name,
+                            "type": "string",
+                            "friendly_name": name}
+                transformed_columns.append(new_json)
+        # Transfored items into rows.
+        if 'items' in data:
+            rows = data['items']
+
+        redash_json_data = {"columns": transformed_columns,
+                            "rows": rows}
+        return redash_json_data
+
+    def api_call(self, sql):
+        # JWT encoded header
+        header = self.generate_header(self.configuration.get('key'),
+                                      self.configuration.get('secret'))
+
+        # URL form using API key file based on GLOBAL
+        url = ("%s/public/api/customers/%s/query" %
+               (self.configuration.get('url'),
+                self.configuration.get('customer_id')))
+
+        # post data base sql
+        post_data_json = {"query": sql}
+
+        response = requests.post(url, headers=header, json=post_data_json,
+                                 verify=self.configuration.get('SSL'))
+
+        if response.status_code == 200:
+            response_output = json.loads(response.content)
+        else:
+            error = 'status_code ' + str(response.status_code) + '\n'
+            error = error + "failed to connect"
+            json_data = {}
+            return json_data, error
+        # if we get right status code then call transfored_to_redash
+        json_data = self.transformed_to_redash_json(response_output)
+        error = None
+        # if we got error from Uptycs include error information
+        if 'error' in response_output:
+            error = response_output['error']['message']['brief']
+            error = error + '\n' + response_output['error']['message']['detail']
+        return json_data, error
+
+    def run_query(self, query, user):
+        data, error = self.api_call(query)
+        json_data = json_dumps(data)
+        logger.debug("%s", json_data)
+        return json_data, error
+
+    def get_schema(self, get_stats=False):
+        header = self.generate_header(self.configuration.get('key'),
+                                      self.configuration.get('secret'))
+        url = ("%s/public/api/customers/%s/schema/global" %
+               (self.configuration.get('url'),
+                self.configuration.get('customer_id')))
+        response = requests.get(url, headers=header, verify=False)
```

Comment:
You need to use the verify configuration value here.

Reply:
Ah! I missed this one. Fixed